homeassistant/components/august/sensor.py

```diff
@@ -226,15 +226,15 @@ def extra_state_attributes(self):
 
         return attributes
 
-    async def async_added_to_hass(self):
+    async def async_added_to_hass(self) -> None:
         """Restore ATTR_CHANGED_BY on startup since it is likely no longer in the activity log."""
         await super().async_added_to_hass()
 
         last_state = await self.async_get_last_state()
         if not last_state or last_state.state == STATE_UNAVAILABLE:
             return
 
-        self._attr_state = last_state.state
+        self._attr_native_value = last_state.state
```

Comment:
Subclasses of SensorEntity should not set `self._attr_state`
I have adjusted to set `self._attr_native_value` instead, but hopefully a code-owner can confirm this is correct.

Reply:
Looks right to me.